profile/README.md

```diff
@@ -13,19 +13,16 @@
 
 ![KOSS Cover](https://imgur.com/xKFP30Y.png)
 
-# Who are We?
+# Who are we?
 
-* We are a bunch of Open Source enthusiasts, who love to share the culture of OSS. Through events and workshops, we try to share the different aspects of Software Engineering, and onboard folks with the basic understanding and literacy of Software Development.
+* We are a bunch of Open Source enthusiasts, who love to share the culture of programming and OSS. Through events and workshops, we try to share the different aspects of Software Engineering.
   
-* We love to work on new ideas and projects, though we don't promote it from KOSS by default. You can find involvement of KOSS members in MetaKGP, developing on a project which is helpful for the KGP Community. You can also find members who are motivated enough to work on funky ideas and projects, which might range to solve a real-world issue, to create a project for banter, to help out folks learn a new language or framework.
+* While we don't work on project ideas as a group, you can find KOSS members involved in MetaKGP, developing a project helpful to the KGP Community. You can also find members who are motivated to work on funky ideas and projects, which might range from solving a real-world issue, to creating a project for banter or helping out folks learn a new language or framework.
   
-* We inculcate an environment, where each of us work on anything that we might want to learn, and in return you would get fantastic mentors on the subject matter. Want to learn Frontend? Hop on to a call with [xypnox](https://github.com/xypnox); Want to learn more about Shell Scripting? [grapheo12](https://github.com/grapheo12) is informative to teach you. Infact, each one of us have carved their own specialization and have had their fair share of experience to guide you, so the KOSS members are the goto folks.
+* We nurture an environment where each of us can work on anything that we might want to learn, and in turn, become a fantastic mentor on the subject. Want to learn Frontend? Hop on to a call with [xypnox](https://github.com/xypnox); Want to learn more about Shell Scripting? Ping [grapheo12](https://github.com/grapheo12) is informative to teach you. In fact, each one of us has carved their own specialization and has had their fair share of experience to guide you, so KOSS members are the goto folks.
 
-* We value culture over technical prowess; You might be a rockstar developer, and quite adept with your skills, but we love to host people, who are empathic, a team member whom you can trust upon, and whom you can depend on. KOSS provides ample of mentoring to folks who want to curate their skills in a field of their choice, and in return they are expected to pass the baton. People having a sound skills is a bonus, but we value your soft-skills more than your technical knowledge.
   
 ## What do we do?  
```

Comment:
If feel like this part has already been touched upon in the part above, and can we removed. What do you think?

Reply:
Agreed, will remove it.